_data-prepper/logs.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: default
+title: Log4j configuration
+nav_order: 12
+---
+
+#Log4j configuration
+
+The following describes how Data Prepper performs logging.
+
+# Logging 
+Data Prepper uses [SLF4J](http://www.slf4j.org/) with a [Log4j 2 binding](http://logging.apache.org/log4j/2.x/log4j-slf4j-impl/). 
+
+For Data Prepper 2.0 or above, Log4j 2 configuration file can be found and edited in `config/log4j2.properties` in the application's home directory. Default properties for Log4j 2 can be found in `log4j2-rolling.properties` in the *shared-config* directory.
```

Comment:
"For Data Prepper versions 2.0 and higher ..." (See "above" in TERMS.md)
"... the Log4j 2 configuration file can be ..."